docs/usage/openapi/ui_plugins.rst

```diff
@@ -0,0 +1,268 @@
+OpenAPI UI Plugins
+------------------
+
+OpenAPI UI Plugins are designed to allow easy integration with your OpenAPI UI framework of choice. These plugins
+facilitate the creation of interactive, user-friendly API documentation, making it easier for developers and end-users
+to understand and interact with the API.
+
+In addition to serving a default JSON representation of the OpenAPI specification, built-in UI Plugins are available to
+support a range of popular OpenAPI documentation tools, including:
+
+- `Scalar <https://scalar.com/>`_
+- `RapiDoc <https://rapidocweb.com/>`_
+- `ReDoc <https://redocly.com/>`_
+- `Stoplight Elements <https://stoplight.io/open-source/elements>`_
+- `Swagger UI <https://swagger.io/tools/swagger-ui/yy>`_
+- `YAML <https://yaml.org/>`_ (OK, not a UI framework, but configured the same way!)
```

Comment:
Missing JSON maybe?

Reply:
The JSON plugin is non-optional.

Reply:
I can include it if you want but the way it works is that if a plugin isn't already registered on the that serves at `/openapi.json`, then we add it - so if an OpenAPIConfig is provided for the application `openapi.json` will always be there.

Reply:
related: https://github.com/litestar-org/litestar/pull/2893#discussion_r1437898435

Reply:
ahh i remember now, ty